Makefile

```diff
@@ -149,11 +149,11 @@ libopenmpt:
 libpsgplay:
 	mkdir -p build
 	cd build && \
-	git -C "psgplay" pull || git clone -b emscripten https://github.com/frno7/psgplay.git "psgplay" 
+	git -C "psgplay" pull || git clone https://github.com/frno7/psgplay.git "psgplay" 
```

Comment:
When cloning, one will want to do `--recurse-submodules`, as [TOS/libc](https://github.com/frno7/toslibc) now is a separate repo and a Git submodule rather than a directory `lib/toslibc`. Alternatively, `git submodule update --init --recursive` and so on. By the way, I tagged `v0.4` having the `web` target, so that’s a stable starting point.